package.json

```diff
@@ -119,7 +119,7 @@
     "build-storybook": "build-storybook --no-dll",
     "lint": "eslint .",
     "lint:fix": "eslint --fix --ext .js,.jsx .",
-    "release": "standard-version"
+    "release": "standard-version --tag-prefix="
```

Comment:
release drafter is already creating tags. This will skip tags for standard release and hopefully grab the latest tag.